spec/models/line_item_spec.rb

```diff
@@ -61,4 +63,63 @@
       end
     end
   end
+
+  context '#value_per_line_item' do
+    subject { line_item.value_per_line_item }
+
+    describe 'item has no value' do
+      it { is_expected.to eq(0) }
+    end
+
+    describe 'item has value and quantity' do
+      let(:value) { 5 }
+      let(:quantity) { 5 }
+
+      before do
+        line_item.item = create(:item, value_in_cents: value)
+        line_item.quantity = quantity
+      end
+
+      it { is_expected.to eq(value * quantity) }
```

Comment:
I am aware that different repos/devs have different thoughts about writing specs; and I'm aware that these could just have easily been written another way, for example:

```ruby
describe 'item has value and quantity' do
  value = 5
  quantity = 5
  line_item.item = create(item: value_in_cents: value)
  line_item.quantity = quantity

  expect(line_item.value_per_line_item).to eq(value * quantity)
end
```

if there is a specific style y'all prefer here, please let me know.